addOns/exim/src/main/java/org/zaproxy/addon/exim/ExtensionExim.java

```diff
@@ -68,6 +72,18 @@ public void hook(ExtensionHook extensionHook) {
         extensionHook.addApiImplementor(new ImportExportApi());
     }
 
+    public void importHarFile(File file) {
+        new HarImporter(file);
+    }
+
+    public void importUrlFile(File file) {
+        new UrlsImporter(file);
+    }
+
+    public void processInput(File file, LogsImporter.LogType logChoice) {
+        new LogsImporter(file, logChoice);
+    }
+
```

Comment:
This seems to be an indirection, given the existing places are doing the imports directly.

Reply:
Yes right
I thought of giving it a layer of abstraction, so new functions can use it.
So, should I import directly too? Or maybe these can be used when new development is done.

Reply:
They can't be used as is though, it's not returning the success status nor allowing to add a progress listener. I'd prefer import directly and refactor later (if really needed).